standalone/pom.xml

```diff
@@ -16,7 +16,7 @@
     <properties>
         <version.guava>18.0</version.guava>
         <version.cassandra.datastax.driver.core>3.4.0</version.cassandra.datastax.driver.core>
-        <version.netty>4.0.47.Final</version.netty>
+        <version.netty>4.1.42.Final</version.netty>
```

Comment:
Is there a reason for the Netty version bump?

Reply:
No specific reason. There was another pull request which contains the same this change, I want to put the same in my PR to make sure it won't create any conflict. I will remove it my next commit.